examples/notebooks/exponential_smoothing.ipynb

```diff
@@ -0,0 +1,483 @@
+    "ExecuteTime": {
+     "end_time": "2017-12-06T09:07:54.181087Z",
+     "start_time": "2017-12-06T09:07:53.395044Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "fit1 = SimpleExpSmoothing(livestock2).fit()\n",
+    "fcast1 = fit1.forecast(9).rename(\"SES\")\n",
+    "fit2 = Holt(livestock2).fit()\n",
+    "fcast2 = fit2.forecast(9).rename(\"Holt's\")\n",
+    "fit3 = Holt(livestock2, exponential=True).fit()\n",
+    "fcast3 = fit3.forecast(9).rename(\"Exponential\")\n",
+    "fit4 = Holt(livestock2, damped=True).fit(phi=0.98)\n",
+    "fcast4 = fit4.forecast(9).rename(\"Additive Damped\")\n",
+    "fit5 = Holt(livestock2, exponential=True, damped=True).fit()\n",
+    "fcast5 = fit5.forecast(9).rename(\"Multiplicative Damped\")\n",
+    "\n",
+    "ax = livestock2.plot(color=\"black\", marker=\"o\", figsize=(12,8))\n",
+    "livestock3.plot(ax=ax, color=\"black\", marker=\"o\", legend=False)\n",
+    "fcast1.plot(ax=ax, color='red', legend=True)\n",
+    "fcast2.plot(ax=ax, color='green', legend=True)\n",
+    "fcast3.plot(ax=ax, color='blue', legend=True)\n",
+    "fcast4.plot(ax=ax, color='cyan', legend=True)\n",
+    "fcast5.plot(ax=ax, color='magenta', legend=True)\n",
+    "ax.set_ylabel('Livestock, sheep in Asia (millions)')\n",
+    "plt.show()\n",
+    "print('Figure 7.5: Forecasting livestock, sheep in Asia: comparing forecasting performance of non-seasonal methods.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2017-10-05T09:40:15.958575Z",
+     "start_time": "2017-10-05T09:40:15.615Z"
+    }
+   },
+   "source": [
+    "## Holt's Winters Seasonal\n",
+    "Finally we are able to run full Holt's Winters Seasonal Exponential Smoothing  including a trend component and a seasonal component.\n",
+    "Statsmodels allows for all the combinations including as shown in the examples below:\n",
+    "1. ```fit1``` additive trend, additive seasonal of period ```season_length=4``` and the use of a Boxcox transformation.\n",
+    "1. ```fit2``` additive trend, multiplicative seasonal of period ```season_length=4``` and the use of a Boxcox transformation..\n",
+    "1. ```fit3``` additive damped trend, additive seasonal of period ```season_length=4``` and the use of a boxcox transformation.\n",
+    "1. ```fit4``` additive damped trend, multiplicative seasonal of period ```season_length=4``` and the use of a boxcox transformation.\n",
+    "\n",
+    "The plot shows the results and forecast for ```fit1``` and ```fit2```.\n",
+    "The table allows us to compare the results and parameterizations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2017-12-06T09:09:03.149848Z",
+     "start_time": "2017-12-06T09:08:54.296071Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "fit1 = ExponentialSmoothing(aust, seasonal_periods=4, trend='add', seasonal='add', use_boxcox=True).fit()\n",
```

Comment:
incorrect usage
`use_boxcox` has no effect here.
It took me a while to realize that it has to be a fit keyword.

Reply:
removed the **kwargs should ensure we don't let this through in future